pkg/options/job-creator.go

```diff
@@ -84,6 +87,16 @@ func CheckJobCreatorOptions(options jobcreator.JobCreatorOptions) error {
 		return err
 	}
 
+	if options.Offer.InputsPath != "" {
+		fileInfo, err := os.Stat(options.Offer.InputsPath)
+		if err != nil {
+			return fmt.Errorf("inputs path error: %w", err)
+		}
+		if !fileInfo.IsDir() {
+			return fmt.Errorf("inputs path must be a directory: %s", options.Offer.InputsPath)
```

Comment:
Maybe we add some logging here unless you already have it upstream

Reply:
These errors will be fatal and dumped when the program exits here:

https://github.com/Lilypad-Tech/lilypad/blob/62253a1bc79d81d87212a7932f52ee79de3736dd/cmd/lilypad/run.go#L33-L36

That seem alright?

Reply:
Sure, let's go with that and then fix if/when we need to